motioneye/remote.py

```diff
@@ -271,7 +272,7 @@ async def set_config(local_config, ui_config) -> Union[str, None]:
         password,
         p,
         method='POST',
-        data=ui_config,
+        data=ui_config.encode('utf-8'),
```

Comment:
```suggestion
        data=ui_config.encode(),
```
UTF-8 is the default.